test/CMakeLists.txt

```diff
@@ -26,6 +26,8 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+file(COPY ${CMAKE_SOURCE_DIR}/Schemas/ DESTINATION ${CMAKE_BINARY_DIR}/Schemas)
```

Comment:
It could be worth copying this directory in the legacy functional test dir too (or instead). And if so, it should be done in `functional-tests-legacy/CMakeLists.txt`.